src/filebrowser/listing.ts

```diff
@@ -1147,15 +1142,11 @@ class DirListing extends Widget {
    * Copy the selected items, and optionally cut as well.
    */
   private _copy(): void {
-    this._clipboard = []
+    this._clipboard = [];
     for (var item of this._getSelectedItems()) {
-      let row = arrays.find(this._items, row => {
-        let text = utils.findElement(row, ITEM_TEXT_CLASS);
-        return text.textContent === item.name;
-      });
       if (item.type !== 'directory') {
         // Store the absolute path of the item.
-        this._clipboard.push('/' + item.path)
+        this._clipboard.push('/' + item.path);
```

Comment:
Possibly, yea, but i'll leave changes like that for another PR, this is just cleanup